mtgjson5/providers/tcgplayer.py

```diff
@@ -333,11 +333,15 @@ def generate_mtgjson_sealed_product_objects(
             sealed_product.identifiers.tcgplayer_product_id = str(product["productId"])
             sealed_product.release_date = product["presaleInfo"].get("releasedOn")
 
-            sealed_product.category = sealed_product.determine_mtgjson_sealed_product_category(
-                sealed_product.name.lower()
+            sealed_product.category = (
```

Comment:
This file should probably be left as it was, unless there's a change I can't tell